frontends/web/src/app.jsx

```diff
@@ -162,13 +156,15 @@ export class App extends Component {
                             // showCreate={true} // Does not exist!
                             // deviceIDs={deviceIDs} // Does not exist!
                         />
-                        <Device
+                        <DeviceSwitch
                             path="/device/:deviceID"
-                            deviceIDs={deviceIDs} />
-                        <Device
+                            key={devices}
```

Comment:
I picked apart multiple commits and this belonged to a different one afaik. Anyway, it carries over here:

https://github.com/digitalbitbox/bitbox-wallet-app/blob/75ae7661a2afa7882bc4e6fd9bb25ddf7303b9f5/frontends/web/src/app.jsx#L167

Needed to that the component remounts with different keys, to trigger the componentDidMount().

Reply:
Can do. After doing the research on resetting components, using the key seems to be a normal solution, but noted.

Didn't use componentWillReceiveProps because it's use seems discouraged:

https://reactjs.org/docs/react-component.html#unsafe_componentwillreceiveprops

Reply:
You can also use https://reactjs.org/docs/react-component.html#componentdidupdate, if you prefer. And I don't know to what degree Preact will follow such deprecations.